### DISCUSSION TITLE
Is there any way to use overlays in viewer.addTiledImage + relative positioning to the tiled image?

### DISCUSSION THREAD

**Discussion by liveman1** [openseadragon/openseadragon#2093] [Q&A] [answered]
Hi,

is it possible to add overlays together with new tiled images using viewer.addTiledImage?

Something like:
```
	viewer.addTiledImage({
      	  tileSource: [{
              url: 'xxx.dzi',
	      x: 0.5,
	      y: 0.2,
	      width: 0.1
              },
          overlays: [{
             id: 'overlay1',
	     x: 0.5,
	     y: 0.2,
	     width: 200,
	     height: 200,
	     className: 'class1'
             },
	     {
             id: 'overlay2',
             x: 0.1,
	     y: 0.7,
	     width: 200,
	     height: 200,
	     className: 'class2'
             }]   
             ]
          }); 
```

If so, is there any way to have a relative positioning of the overlay within the added tiled image? This would allow not to touch the overlay coordinates if the coordinates of the tiles image change.
For example the relative coordinates of overlay1 should be `x: 0.0, y: 0.0`,

Thanks a lot for your help!

**Comment by iangilman** [comment, marked as answer]
OSD does not support tiled image-relative overlays. It's certainly possible we could add such a thing, but it would probably be a bit of work. Until that feature exists, there's probably no point to adding an `overlays` option to `addTiledImage`. Instead you can use `addOverlay`. 

If you're interested in pursuing tiled image-relative overlays, feel free to file an issue for it!

**Reply by liveman1** [reply]
Thanks!  :)
I will open an issue. This would be a nice enhancement of OSD.
